package.json

```diff
@@ -2,15 +2,20 @@
   "name": "flat-camel",
   "version": "1.0.0",
   "description": "flattener module",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
-    "test": "npm test"
+    "test": "mocha test"
   },
   "keywords": [
     "node",
     "flattener",
     "camel"
   ],
   "author": "Dial-Once",
-  "license": "ISC"
+  "license": "ISC",
+  "devDependencies": {
+    "eslint": "^4.4.1",
+    "flatten-obj": "^3.1.1",
```

Comment:
this should not be a dev dependency since you use it in src/index.js, but a dependecy